service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsMidEventControllerTest.java

```diff
@@ -58,35 +50,29 @@ void shouldAddAmountToPayFieldWhenFeatureToggleIsTrue() {
             .containsEntry("displayAmountToPay", YES.getValue());
     }
 
-    @Test
-    void shouldNotAddAmountToPayFieldWhenFeatureToggleIsFalse() {
-        given(ldClient.boolVariation(eq("FNP"), any(), anyBoolean())).willReturn(false);
-
-        AboutToStartOrSubmitCallbackResponse response = postMidEvent(CaseDetails.builder()
-            .data(Map.of("c2ApplicationType", Map.of("type", "WITH_NOTICE")))
-            .build(), "get-fee");
-
-        verify(feeService, never()).getFeesDataForC2(WITH_NOTICE);
-        assertThat(response.getData()).doesNotContainKeys("amountToPay", "displayAmountToPay");
-    }
-
     @Test
     void shouldRemoveTemporaryC2DocumentForEmptyUrl() {
-        given(ldClient.boolVariation(eq("FNP"), any(), anyBoolean())).willReturn(false);
+        given(feeService.getFeesDataForC2(WITH_NOTICE)).willReturn(FeesData.builder()
+            .totalAmount(BigDecimal.TEN)
+            .build());
 
         AboutToStartOrSubmitCallbackResponse response = postMidEvent(CaseDetails.builder()
-            .data(Map.of("temporaryC2Document", Map.of("document", Map.of())))
+            .data(Map.of("temporaryC2Document",
+                Map.of("document", Map.of()),"c2ApplicationType", Map.of("type", "WITH_NOTICE")))
             .build(), "get-fee");
 
         assertThat(response.getData()).extracting("temporaryC2Document").extracting("document").isNull();
     }
 
     @Test
     void shouldKeepTemporaryC2DocumentForNonEmptyUrl() {
-        given(ldClient.boolVariation(eq("FNP"), any(), anyBoolean())).willReturn(false);
+        given(feeService.getFeesDataForC2(WITH_NOTICE)).willReturn(FeesData.builder()
```

Comment:
My understanding is, since we removed fee toggle, now all fee mid-event tests need fee mock. All tests seems valid to me now but appreciate if someone could check again for me.